components/Footer.tsx

```diff
@@ -32,7 +32,7 @@ const Footer = () => {
               <img src={Twitter} />
             </a>
           </SocialLinks>
-          <Copyright>HackSC 2019 &copy;</Copyright>
+          <Copyright>HackSC 2020 &copy;</Copyright>
```

Comment:
Might as well dynamically render date:
```suggestion
          <Copyright>HackSC {(new Date()).getFullYear()} &copy;</Copyright>
```